engine/class_modules/sc_shaman.cpp

```diff
@@ -372,6 +380,13 @@ struct shaman_t : public player_t
     cooldown_t* strike;  // shared CD of Storm Strike and Windstrike
   } cooldown;
 
+  // Covenant Class Abilities
+  struct
+  {
+    // Necrolord
+    const spell_data_t* necrolord; // Primordial Wave
```

Comment:
This naming convention is what I saw used in the Druid module so I ran with that.